### PR TITLE
Updating Manual Install Instructions Based on Feedback from slangerx

### PR DESCRIPTION
**Pull request by seth-shaw-asu**
## Purpose / why
@slangerx contributed edits to the manual install instructions and posted it as an issue @ https://github.com/Islandora/documentation/issues/2326. This PR pulls in most of those suggestions and provides a few small additions.

## What changes were made?

Lots of updates to the various manual install steps.

## Verification
Please check formatting, spelling, grammar, etc. 

## Interested Parties

* @Islandora/documentation, esp @ysuarez 
* @Islandora/committers

### Pull-request Reviewer
Pull-request reviewer should ensure the following:

* [✔️ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

### Person Merging
The person merging should ensure the following:

* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

File: docs/installation/manual/installing-alpaca.md [hunk 1]
```diff
@@ -32,15 +34,69 @@ sudo apt-cache policy activemq
...
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Update the WebConsolePort host property settings in `/opt/activemq/conf/jetty.xml` from `<property name="host" value="127.0.0.1"/>` to `<property name="host" value="0.0.0.0"/>` so that you can access the dashboard from outside the local machine.
+
+Optionally, change the dashboard user credentials in `/opt/activemq/conf/users.properties`.
+
+*Note*
```
Comment (rosiel) [inline docs/installation/manual/installing-alpaca.md]: note format is
```
!!! note "Helpful Tip"
    i am a helpful tip
```
i.e. all note text lines need to be indented 4 spaces.

**Comment by seth-shaw-asu** [comment]
Should be ready to go now.

**Review by rosiel — APPROVED** [review]

File: docs/installation/manual/installing-solr.md [hunk 2]
```diff
@@ -66,33 +67,12 @@ Then apply your new configuration.
...
-```
-- `SOLR_CORE`: `islandora8`
-
-You should see an output similar to this:
-```
-WARNING: Using _default configset with data driven schema functionality. NOT RECOMMENDED for production use.
-         To turn off: bin/solr config -c islandora8 -p 8983 -action set-user-property -property update.autoCreateFields -value false
-
-Created new core 'islandora8'
-```
```
Comment (forresthopkinsa) [inline docs/installation/manual/installing-solr.md:88]: Why was this removed? The rest of the page refers to "the core we created earlier" but there is no core now

Comment (forresthopkinsa) [reply docs/installation/manual/installing-solr.md:88]: Update: it looks like this was just moved down to line 134. IMO this needs to be updated to be in the right order. I can make this change without objection

File: docs/installation/manual/installing-crayfish.md [hunk 3]
```diff
@@ -224,74 +251,38 @@ security:
...
+Edit `/opt/crayfish/Milliner/config/packages/monolog.yaml` to point to the new logging directory:
+
+```yml
+        milliner:
+            type: rotating_file
+            path: /var/logs/islandora/Milliner.log
 ```
 
+Edit the commons config to update it with Fedora's location (if necessary) and enable the apix middleware in `/opt/crayfish/Milliner/config/packages/crayfish_commons.yaml`:
+
```
Comment (forresthopkinsa) [inline docs/installation/manual/installing-crayfish.md:285]: ?

File: docs/installation/manual/installing-solr.md (hunk 2 above)
Comment (seth-shaw-asu) [reply docs/installation/manual/installing-solr.md:88]: @forresthopkinsa , this PR has been merged already, but if you would like to propose any edits, you can do it on https://github.com/Islandora/documentation/blob/main/docs/installation/manual/installing-solr.md
